auth-server/src/main/resources/application.properties

```diff
@@ -1,14 +1,22 @@
+#database
+spring.datasource.url = jdbc:postgresql://localhost:5432/MyFirstDB
+spring.datasource.username = postgres
+spring.datasource.password = admin
+spring.datasource.driverClassName=org.postgresql.Driver
+spring.jpa.hibernate.ddl-auto=create
+spring.jpa.show-sql=true
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
+
```

Comment:
Are this Configurations needed ?

Reply:
no,left off from previous projects will, remove